packages/openneuro-components/src/activity-slider/ActivitySlider.tsx

```diff
@@ -1,96 +1,139 @@
+        containerClass={containerClass}
+        itemClass={itemClass}
+        showDots={showDots}
+        responsive={responsive}
+        customLeftArrow={<LeftArrow />}
+        customRightArrow={<RightArrow />}>
+        {data.map(({ node }) => (
+          <div className="activity-slider-node" key={node.id}>
+            <div className="ds-name">
+              <h4>
+                <Link
+                  to={
+                    'datasets/' +
+                    node.id +
+                    '/versions/' +
+                    node.latestSnapshot.tag
+                  }>
+                  {node.latestSnapshot.description.Name}
+                </Link>
+              </h4>
+            </div>
 
-                {item.node.publishDate ? (
-                  <div className="ds-pub-date">
-                    {formatDistanceToNow(parseISO(item.node.publishDate))} ago
-                  </div>
-                ) : null}
+            {node.publishDate ? (
+              <div className="ds-pub-date">
+                {formatDistanceToNow(parseISO(node.publishDate))} ago
+              </div>
+            ) : null}
 
-                {item.node.analytics ? (
-                  <div className="ds-pub-views">
-                    {item.node.analytics.views.toLocaleString()} views
-                  </div>
-                ) : null}
+            {node.analytics ? (
+              <div className="ds-pub-views">
+                {node.analytics.views.toLocaleString()} views
+              </div>
+            ) : null}
 
-                <div className="ds-modality">
-                  <div className="hexagon-wrapper">
-                    {item.node.latestSnapshot.summary !== null ? (
-                      <>
-                        <div
-                          className={
-                            'hexagon ' +
-                            item.node.latestSnapshot.summary.modalities[0].substr(
-                              0,
-                              4,
-                            )
-                          }></div>
-                        <div className="label">
-                          {item.node.latestSnapshot.summary.modalities[0].substr(
-                            0,
-                            4,
-                          )}
-                        </div>
-                      </>
-                    ) : (
-                      <>
-                        <div className="hexagon no-modality"></div>
-                        <div className="label">N/A</div>
-                      </>
-                    )}
-                  </div>
-                </div>
+            <div className="ds-modality">
+              <div className="hexagon-wrapper">
+                {node.latestSnapshot.summary !== null ? (
+                  <>
+                    <div
+                      className={
+                        'hexagon ' +
+                        node.latestSnapshot.summary.modalities[0].substr(0, 4)
```

Comment:
This is probably good for now but we should add a primaryModality field.

Reply:
yeah this is also used on the dataset page. I added a TODO for later.